platform/search/test/GenericSearchWorkerSpec.js

```diff
@@ -0,0 +1,132 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define,describe,it,expect,runs,waitsFor,beforeEach,jasmine,Worker*/
+
+/**
+ *  SearchSpec. Created by shale on 07/31/2015.
+ */
+define(
+    [],
+    function (GenericSearchWorker) {
+        "use strict";
+
+        describe("The generic search worker ", function () {
+            // If this test fails, make sure this path is correct
+            var worker = new Worker('platform/search/src/GenericSearchWorker.js'),
```

Comment:
To do this with a relative path, first, require "require" in your define call, and the require argument that you'll receive has a "toUrl" function which can be used to resolve relative paths, so you can change this line to:

```javascript
var worker = new Worker(require.toUrl('../src/GenericSearchWorker.js')),
```

From the RequireJS API docs: http://requirejs.org/docs/api.html#modulenotes-urls